lib/core/utils.js

```diff
@@ -109,6 +109,17 @@ RESUtils.isMatchURL = function(moduleID) {
 
 	return included;
 };
+RESUtils.minifyOptions = function(obj) {
+	var min = {};
+	if (obj) {
+		for (var key in obj) {
+			if ('value' in obj[key]) {
+				min[key] = {value: obj[key].value};
+			}
+		}
+	}
+	return min;
+};
```

Comment:
I see a few instances of `RESStorage.getItem('RESoptions.` or `.setItem('RESoptions.`:

* `bitcointip.load` and `.save`, but IIRC that was only used in the standalone build. :white_check_mark: 
* `commentTools.migrateData`: will dump unminified options directly into storage, but they'll be minified next time options are saved. it only affects commentTools and commentPreview options, so not a big concern.  :white_check_mark: however, this ought to be refactored into migrate.js at some point. :warning:
* `filteReddit.checkFilterFormat`: stores unminified settings every pageload if the user has any subreddit filters. worth minifying. :exclamation:  also shouldn't re-set settings every pageload, only if anything changed! :warning: